pyte/screens.py

```diff
@@ -926,7 +927,7 @@ def select_graphic_rendition(self, *attrs):
                 attr = g.TEXT[attr]
                 replace[attr[1:]] = attr.startswith("+")
             elif not attr:
-                replace = self.default_char._asdict()
+                replace = self.default_char_dict
```

Comment:
Yes, the frequency of calling this method is high, and get data from memory much faster than calculate it.
Before:
[lineprof_asdict.txt](https://github.com/selectel/pyte/files/983257/lineprof_asdict.txt)
After:
[lineprof_memory_dict.txt](https://github.com/selectel/pyte/files/983260/lineprof_memory_dict.txt)